libraries/AP_TECS/AP_TECS.cpp

```diff
@@ -1029,10 +1092,20 @@ void AP_TECS::_update_pitch(void)
     const float SEBdot_dem_max = _maxClimbRate * GRAVITY_MSS;
     if (SEBdot_dem < SEBdot_dem_min) {
         SEBdot_dem = SEBdot_dem_min;
-        _SEBdot_dem_clip = clipStatus::MIN;
+        // The vertical accln control method is incompatible with the height demand
+        // profile mechanism controlled by _SEBdot_dem_clip
+        if (!option_is_set(Option::HGT_ACCLN_CTRL) &&
+            !option_is_set(Option::INHIBIT_CLIPSTATUS)) {
+            _SEBdot_dem_clip = clipStatus::MIN;
+        }
```

Comment:
If I read this correctly, you are preventing clipping in the climb/descent rate here: The only consumer of `_SEBdot_de_clip` is `_update_height_demand()`.
This is the only modification in `_update_pitch()`, right?

Then my question is, why run `_update_pitch()` at all? Its result is not used during acceleration control mode.